### PR TITLE
Added .zenodo.json file with instructions.

### PR DESCRIPTION
**Pull request by chrisfilo**
When releasing remember to:

- Use a consistent release title with the version number
- Populate the release description with the changelog

### CODE REVIEW

**Comment by effigies** [comment]
Do we want to hook in Github IDs, as well? Might make it easier to assess whether we've gotten everybody.

**Comment by chrisfilo** [comment]
I don't think Zenodo metadata supports that.

**Comment by effigies** [comment]
Gotcha. I assumed this was your own json construction.

**Comment by chrisfilo** [comment]
Oh no this is Zenodos brainchild. After merging this each time we make a
release Zenodo will mint a new DOI with all the proper names, affiliations
and orcids.

On Apr 1, 2017 5:55 AM, "Chris Markiewicz" <notifications@github.com> wrote:

Gotcha. I assumed this was your own json construction.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/nipy/nipype/pull/1924#issuecomment-290917552>, or mute
the thread

**Comment by effigies** [comment]
I'd previously done a little digging to find users and affiliations that aren't in the Zenodo so my BibTeX didn't look so ugly. Here are my best estimates as to some missing contributors:

- [Carlo de los Angeles](https://github.com/nipy/nipype/commits?author=cdla) (@cdla) - Affiliation: Gabrieli Lab (MIT) at time of commits; Etkin Lab (Stanford) presently
- [Fred Loney](https://github.com/nipy/nipype/commits?author=fredloney) (@FredLoney) - Affiliation: OHSU -  Possible OrcID: 0000-0002-8828-0726
- [Ben Cipollini](https://github.com/nipy/nipype/commits?author=bcipolli) (@bcipolli) - Affiliation: UCSD - OrcID: 0000-0002-7782-0790
- [Drew Erickson](https://github.com/nipy/nipype/commits?author=drewerickson) (@drewerickson) - Affiliation: Stanford
- [Jason Wong](https://github.com/nipy/nipype/commits?author=jason-wg) (@jason-wg)
- [Brendan Moloney](https://github.com/nipy/nipype/commits?author=moloney) (@moloney) - Affiliation: Advanced Imaging Research Center, Oregon Health Science University
- [Steven Giavasis](https://github.com/nipy/nipype/commits?author=sgiavasis) (@sgiavasis) - Affiliation: Child Mind Institute
- [Mathias Goncalves](https://github.com/nipy/nipype/commits?author=mgxd) (@mgxd) - Affiliation: MIT

This is an incomplete list based off of GitHub handles in the previous Zenodo releases. I've included these handles, so hopefully they'll see this and alert us if I got anything wrong.

File: .zenodo.json [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+        "orcid": "0000-0001-5194-6056"
+      }, 
+      {
+        "affiliation": "University College London", 
+        "name": "Eshaghi, Arman", 
+        "orcid": "0000-0002-6652-3512"
+      }, 
+      {
+        "affiliation": "Boston University", 
+        "name": "Markiewicz, Christopher",
```
Comment (effigies) [inline .zenodo.json]: Would prefer "Markiewicz, Christopher J." for citations.

**Comment by chrisfilo** [comment]
Thanks for digging this up. I modified your initial and I will merge it as it is.

Since I do not want to force anyone on the publication or assume what their pen names and affiliations should be I would appreciate if the missing mentioned people send a PRs modifying the .zenodo.json.

**Pull request merged**